lib/utils/protocols/rtsp.ts

```diff
@@ -87,19 +87,20 @@ export const sessionTimeout = (buffer: Buffer) => {
    * session-id        =  1*256( ALPHA / DIGIT / safe )
    * delta-seconds     =  1*19DIGIT
    */
+  const defaultTimeout = 60
```

Comment:
I just want to double check, this is actually 60 seconds and not 60 milliseconds?

Don't you have some kind of constants file you can put this in? Or perhaps you prefer to keep it with it context? I'm fine with it either way.

Is this a breaking change?

Reply:
This shouldn't be a breaking change, just that we will send an OPTIONS every 60s if there was no timeout specified.